operatorinterface.py

```diff
@@ -28,6 +29,14 @@ def invert() -> float:
     return invert
 
 
+def SignSquare(inputFn: AnalogInput) -> AnalogInput:
```

Comment:
What exactly is this for

Reply:
what this does is it squares the input while keeping respect to the sign
it allows finer control at lower speeds -- very much driver preference

see: 
![image](https://user-images.githubusercontent.com/91603936/223606458-fe28a1a6-7583-4103-8a05-db6869491924.png)